src/lib/drivers/cri.js

```diff
@@ -44,9 +44,13 @@ class CriDriver extends Driver {
         chromeRemoteInterface({port: port, chooseTab: tab}, chrome => {
           this._chrome = chrome;
           this.beginLogging();
-          this.enableRuntimeEvents().then(_ => {
-            resolve();
-          });
+
+          // Switch on events from domains we care about.
+          this.enableRuntimeEvents()
+              .then(_ => this.enableSecurityEvents())
```

Comment:
There are two gatherers that need the security events, so what's the order here? They both register their listeners, and I guess they can both enable the security events, but there's a disambiguation issue here: how do you know you've picked up the "most recent" securityChanged event here?